server/src/test/java/org/eclipse/lsp/cobol/core/semantics/outline/RangeUtilsTest.java

```diff
@@ -16,6 +16,7 @@
 package org.eclipse.lsp.cobol.core.semantics.outline;
```

Comment:
You can move the test to `delegates.util` package too.

Reply:
Oops, I missed that.